### PR TITLE
CLA on Gnosis

### PR DESCRIPTION
**Pull request by thedriftofwords**
https://documentation-git-fork-thedriftofwords-aut-fa41db-chainlinklabs.vercel.app/chainlink-automation/overview/supported-networks#gnosis-chain-xdai

https://documentation-git-fork-thedriftofwords-aut-fa41db-chainlinklabs.vercel.app/chainlink-automation/overview/automation-release-notes#2024-03-25---automation-on-gnosis

### CODE REVIEW

File: src/config/data/chains.json [hunk 1]
```diff
@@ -60,8 +84,22 @@
       "GNOSIS_MAINNET": {
         "chainId": 100,
         "title": "Gnosis Chain mainnet",
-        "explorer": "https://blockscout.com/poa/xdai",
-        "nativeCurrency": { "name": "xDAI", "symbol": "XDAI", "decimals": 18 }
+        "explorer": "https://gnosis.blockscout.com/",
```
Comment (friedemannf) [inline src/config/data/chains.json:87]: Can we do `https://gnosisscan.io` here? That's the Etherscan clone and where we're verifying contracts.

File: src/features/chainlink-automation/data/chainlink-automation-config.json [hunk 2]
```diff
@@ -373,5 +373,55 @@
     "stalenessSeconds": 90000,
     "registrar": "0xD8983a340A96b9C2Bb6855E46847aE134Db71fB1",
     "transcoder": "0x395C8461299c9981E9D1A30d116F0c80a927d2A4"
+  },
+  "GNOSIS_MAINNET": {
+    "paymentPremiumPPB": 300000000,
```
Comment (friedemannf) [inline src/features/chainlink-automation/data/chainlink-automation-config.json:378]:
```suggestion
    "paymentPremiumPPB": 1000000000,
```

**Review by friedemannf — CHANGES_REQUESTED** [review]

File: src/config/data/chains.json (hunk 1 above)
Comment (dwightjl) [reply src/config/data/chains.json:87]: chains.json is auto-generated from [chainlist](https://chainlist.org/) data. This is used to set the explorer when developer add LINK to their wallets. It shouldn't apply to URLs in our documentation, but we should double check that.

Comment (thedriftofwords) [reply src/config/data/chains.json]: I manually updated it in this PR anyway

**Review by friedemannf — APPROVED** [review]

**Review by dwightjl — APPROVED** [review]

**Pull request merged**